solidity/scripts/get-default-application-account.js

```diff
@@ -1,5 +1,11 @@
 module.exports = async function () {
   const accounts = await web3.eth.getAccounts()
-  console.log(`${accounts[6]}`)
+  // arbitrary chosen account for default application
+  if (accounts.length < 7) {
+    // Assign the last account address as application account
+    console.log(`${accounts[accounts.length - 1]}`)
+  } else {
+    console.log(`${accounts[6]}`)
+  }
```

Comment:
In the context of this script, `7` is a magic number. 
I think we should consider altering the logic in `solidity/integration/smoke_test.js` and `stress_test.js` to take `accounts[1]` as the application. Leave a comment in this script why it's `accounts[1]` and if `accounts.length == 1` take `accounts[0]` - just one special case. WDYT?

Reply:
`7` was magic because the default application was at `6` which I take was also chosen arbitrary. 

If we introduce more accounts in [docker for Celo](https://github.com/keep-network/keep-core/pull/2324) then maybe we should just revert the current change and leave `smoke_test.js` & `stress_test.js` as is?

Reply:
We will probably still have a problem with Alfajores where we'll always have only one account. We can't skip `get-default-application-account.js` execution for non-local, because then, we'd also have to skip `lcl-client-config.js`.

Having a special case for just one account in the system in this particular case is fine IMO.

Reply:
Both `get-default-application-account.js` & `lcl-client-config.js` are called only from initialize scripts. If we limit the initialize scripts only to `local` then I think we don't have this problem anymore.

Reply:
I'd love to have `lcl-client-config.js` called for non-local. For Alfrajores we can assume contract owner is the application and client configuration such as bonding or staking needs to be done from dashboard.

Reply:
22f7d3c 

Given that we will create more Celo accounts running on docker, I left a default application account as it was implemented originally. IMO current implementation reflects a real scenario a bit better, when a `member` doesn't have the same address as `default application`, ie. both accounts[1]. 

If you think it is better to have them both assigned to `accounts[1]` I can change it.

Reply:
PR with 3 extra accounts: https://github.com/keep-network/keep-core/pull/2332